src/Microsoft.IdentityModel.Xml/DsigSerializer.cs

```diff
@@ -101,37 +101,42 @@ public virtual KeyInfo ReadKeyInfo(XmlReader reader)
                     // <X509Data>
                     if (reader.IsStartElement(XmlSignatureConstants.Elements.X509Data, XmlSignatureConstants.Namespace))
```

Comment:
Should factor into a method that returns an X509Data, it can be private such as ReadIssuerSerial.

Reply:
Done!